File: src/main/java/com/learnershi/rclasssocket/entity/QuizShare.java
package com.learnershi.rclasssocket.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.Accessors;
import org.springframework.data.mongodb.core.mapping.Document;

/**
 * Quiz
 *
 * @author Jae Deok
 */
@Accessors(chain = true)
@NoArgsConstructor
@Getter
@Setter
@Document(collection = "quizShare")
public class QuizShare extends ClassRoomEntity {
    private String userName;
    private String answer;
}

File: src/main/java/com/learnershi/rclasssocket/entity/QuizShare.kt
package com.learnershi.rclasssocket.entity

import org.springframework.data.mongodb.core.mapping.Document

/**
 * Quiz
 *
 */
@Document(collection = "quizShare")
class QuizShare : ClassRoomEntity() {
    private val userName: String? = null
    private val answer: String? = null
}